applications/tari_base_node/windows/start_tari_basenode_fresh.bat

```diff
@@ -0,0 +1,39 @@
+@echo.
+@echo Set up environment variables
+@echo ----------------------------
+@rem This is the basenode executable name
+@set my_exe=tari_base_node.exe
+
+@rem This is the location of the configuration and identity files
+@set config_path=%~dp0config
+
+@rem This is the basenode executable location, or leave blank if in path
+@set my_exe_path=%config_path%
+@rem set my_exe_path=
+
+@rem The base folder where the database and log files will be located
+@set base_path=%~dp0
+@if %base_path:~-1%==\ set base_path=%base_path:~0,-1%
+
+@echo.
+@echo Cleanup old data
+@echo ----------------
+@echo rincewind
+@rd /s /q rincewind
+@echo wallet
+@rd /s /q wallet
```

Comment:
This is really dangerous -- deleting the users' wallets without any indication or warning

Reply:
Agreed, I will remove this one